nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -86,6 +89,25 @@ def _sanitize_k8s_object(cls, data):
         else:
             return data
 
+    @staticmethod
+    def _list_helm_releases():
+        cmd = 'helm list --all-namespaces -o json'
+        try:
+            result = run(cmd, capture_output=True,
+                         timeout=10, encoding='UTF-8',
+                         shell=True, check=False)
```

Comment:
We have an util function (`execute_cmd`) to run command in `nuvlaedge.agent.common.util` which is used almost everywhere in the agent.
I think it would be good to reuse it and improve it as needed (for example by adding a `timeout` argument and/or `**kwargs` that can be passed to `run()` and `Popen()`.
If you want to handle exceptions here, we can add a `raise_exceptions` parameter to `execute_cmd`.

Reply:
JSON parsing could also be added as an option in `execute_cmd`.
Like in `nuvlaedge.common.file_operations.read_file`